packages/mobile/android/app/build.gradle

```diff
@@ -350,6 +350,8 @@ dependencies {
     implementation "com.google.android.gms:play-services-cast-framework:${safeExtGet('castFrameworkVersion', '+')}"
 
     implementation project(':react-native-google-cast')
+    
+    implementation 'com.bytedance.ies.ugc.aweme:opensdk-oversea-external:0.2.0.2'
```

Comment:
@sliptype heads up had to change this line from when i first opened the PR to get the sdk working with android

Reply:
Ok cool, I'm still in ios land so maybe this will be merged by the time i get to android!

Reply:
sweeet yeah will merge as soon as it passes CI so hopefully v soon